hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieAvroHFileReader.java

```diff
@@ -206,7 +198,12 @@ protected ClosableIterator<IndexedRecord> getIndexedRecordIterator(Schema reader
     }
 
     // TODO eval whether seeking scanner would be faster than pread
-    HFileScanner scanner = getHFileScanner(reader, false);
+    HFileScanner scanner = null;
+    try {
+      scanner = getHFileScanner(reader, false, false);
+    } catch (IOException e) {
+      throw new HoodieIOException("Instantiation HfileScanner failed for " + reader.getHFileInfo().toString());
+    }
```

Comment:
The try catch is for logging purpose?

Reply:
every other method in the interface throws IOException except this method. So, I let it as is.